Problem with sptable

Dear All,

As you mentioned in a previous issue, I installed iraf and iraf-sptable using apt on Ubuntu 19.10 but when I tried to use "fxcor" in "xrv" this message appear:
ERROR: Cannot open connected subprocess (./x_xrv.e)
The same happens for other utilities.

That is true. unfortunately. The problem is that the package should have been linked to the "fftpack" library, wich didn't happen. From the [build log](https://launchpad.net/ubuntu/+source/iraf-sptable/1.0~pre20180612-1build2/+build/17365294/+files/buildlog_ubuntu-eoan-amd64.iraf-sptable_1.0~pre20180612-1build2_BUILDING.txt.gz):
```
xc -Nz -p sptable -p noao -o xx_xrv.e x_xrv.o libpkg.a -lasttools -lsmw -ltbtables -lxtools -ldeboor -lcurfit -liminterp -lllsq -lnlfit ../../bin/libex.a
/usr/bin/ld: libpkg.a(numrep.o): in function `realft_':
./src/xrv/numrep.x:28: undefined reference to `rffti_'
/usr/bin/ld: ./src/xrv/numrep.x:46: undefined reference to `rfftb_'
/usr/bin/ld: ./src/xrv/numrep.x:31: undefined reference to `rfftf_'
/usr/bin/ld: libpkg.a(numrep.o): in function `twofft_':
./src/xrv/numrep.x:69: undefined reference to `cffti_'
/usr/bin/ld: ./src/xrv/numrep.x:76: undefined reference to `cfftf_'
/usr/bin/ld: ./src/xrv/numrep.x:77: undefined reference to `cfftf_'
collect2: error: ld returned 1 exit status
Warning, mkpkg line 7: module `relink' not found or returned error
move `xx_xrv.e' to `spbin$x_xrv.e'
$move: file `xx_xrv.e' not found
```
The undefined references are in the fftpack library. I will fix this here ASAP.

--- a/src/xrv/mkpkg
+++ b/src/xrv/mkpkg
@@ -9,7 +9,7 @@ update:
 	;
 
 relink:
-	$set LIBS = "-lasttools -lsmw -ltbtables -lxtools -ldeboor -lcurfit -liminterp -lllsq -lnlfit"
+	$set LIBS = "-lasttools -lsmw -ltbtables -lxtools -ldeboor -lcurfit -liminterp -lllsq -lnlfit -lfftpack"
 	$update libpkg.a
 	$omake x_xrv.x
 	$link -o xx_xrv.e x_xrv.o libpkg.a $(LIBS) ../../bin/libex.a

